src/App.js

```diff
@@ -47,25 +45,24 @@ class App extends Component {
               <Link to='/signin'>
               <button type="button">Login</button>
               </Link>
+            <Link to="/">
             <button
               type="button"
-              onClick={event => this.submitLogout(event)}>Logout</button>
+              onClick={event => this.submitLogout(event)}>Logout
+            </button>
+            </Link>
             <img className='flim-reel' src={Film}/>
-            <Login setUser={this.setUser} />
           </header>
-            <Switch>
-              <Route exact path='/signin' render={() => <Login setUser={this.setUser} /> } />
-              <Route path='/' Component = <Movies /> />
-              // <Route path='/' render={() => <Movies movies={this.state.movies} /> } />
-              // <Route path='/logout' render={() => <App />} />
-            </Switch>
-          {/* <section className='all-cards'>
-            <Movies movies={this.state.movies} />
-          </section> */
-          }
-          <Movies />
-          </Route>
-        </BrowserRouter>
```

Comment:
From the lesson I had we decided to remove this section, we found that the switch was not needed